### DISCUSSION TITLE
New Feature Request: Mute Microphone

### DISCUSSION THREAD

**Discussion by ghost** [File-New-Project/EarTrumpet#642] [Ideas]
Hi,

I have to say I love this tool.  The one thing I wish it had is an easy way to mute the recording device at a layer so that I can be certain that the mic isn't picking up room sounds when I don't want it to.  To more or less disable audio from it.  Some headsets don't have a mute button on them, I unfortunately have one of them.

This also become a great security feature.

Keep up the great work! You Rock!

Regards,
CM

**Comment by iG8R** [comment]
Hi.
The same to me. Please, add this feature - it would be very convenient.

**Comment by kilbasar** [comment]
+1. More broadly, would be great to have default Microphone (and/or other recording devices) as a level you can adjust/mute and showing the current output, just like is done for output sources. Thanks!

**Comment by vertigo220** [comment]
I'd like this as well. I always unplug my mic when not in use for privacy reasons, and while any software disconnection isn't completely secure, it would be a nice option to have to be able to quickly disable and enable the mic this way, especially with a shortcut. Not sure if it's possible, but if it could be made to automatically enable and switch to it when specified software is brought to the foreground and disabled when it's minimized (or maybe there's a better way to automate it), that would be great. So I could have it automatically enabled when I open Discord (not launch, because I always leave it running, so would have to be based on something like a taskbar button showing) and disabled when I minimize or close it. Or perhaps it could detect sounds coming from certain apps, so it would enable the mic when Discord plays a dialing sound or the sound of answering a call, and disconnect it when it plays the sound of hanging up.

**Comment by Xionghaizi001** [comment]
The same to me.
